spec/factories/state_file_az_intakes.rb

```diff
@@ -102,6 +102,7 @@
     primary_last_name { "Onian" }
     primary_birth_date { Date.new((MultiTenantService.statefile.current_tax_year - 65), 12, 1) }
     state_file_analytics { StateFileAnalytics.create }
+    # phone_number: "+15551115511" # number taken from specs for Phony
```

Comment:
I had to dig through `Phony` specs to find a valid number for testing since `555-555-5555` was rejected. I put it where people might expect to find it but I left it in a comment in case tests were depending on phone numbers being nil by default.